nixos/modules/security/acme.nix

```diff
@@ -7,6 +7,11 @@ let
   numCerts = length (builtins.attrNames cfg.certs);
   _24hSecs = 60 * 60 * 24;
 
+  # Used to make unique paths for each cert/account config set
+  mkHash = with builtins; val: substring 0 20 (hashString "sha256" val);
```

Comment:
@m1cr0man What happens if the first 20 characters of the hash are not unique?

Reply:
Cert/account folders have the same name for different certs. Lego creates subdirectories that are unique per domain/account. This is a non issue.